docs/assembly.rst

```diff
@@ -947,11 +980,14 @@ Pseudocode::
       POP
     LabelDefinition(name [id1, ..., idn] :) ->
       JUMPDEST
-      // register new variables id1, ..., idn and set the stack height to
-      // stack_height_at_block_start + number_of_local_variables
+      // register new variables id1, ..., idn and adjust the assumed stack
```

Comment:
Any identifier is "local" to its block in the sense that it is only visible in the block (and its sub-blocks). Variables introduced via labels only differ from variables introduced via `let` in the way their stack slot is "generated".